frontend/src/container/Tracim.jsx

```diff
@@ -247,6 +282,18 @@ export class Tracim extends React.Component {
     return (
       <div className='tracim fullWidthFullHeight'>
         <Header />
+        {state.displayConnectionError &&
+          <FlashMessage
+            id='connection_error'
+            flashMessage={
+              [{
+                message: props.t('Tracim has a connection problem, please wait or reload the page if the problem persists'),
+                type: 'danger'
+              }]
+            }
+            showCloseButton={false}
+            t={props.t}
+          />}
```

Comment:
to have the closing bracket at the same column as the opening one while having standardjs ok with it is by using parenthesis like so
```suggestion
        {state.displayConnectionError && (
          <FlashMessage
            id='connection_error'
            flashMessage={
              [{
                message: props.t('Tracim has a connection problem, please wait or reload the page if the problem persists'),
                type: 'danger'
              }]
            }
            showCloseButton={false}
            t={props.t}
          />
        )}
```